database/src/main/resources/db/migration/V1.89.0__RHCLOUD-29443_remove_secrets_columns.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE
```

Comment:
I think we should move this Drop column script to another PR, and merge it a little bit after, just in case we have to rollback the prod delivery for any reason.

Reply:
You are right. I have removed the migration but we have to coordinate to merge this one, because I need to run the migrations first before removing all the code!